frontend/src/components/sitemanager/calendar.js

```diff
@@ -42,13 +45,66 @@ export default class Example extends React.Component {
     {
       this.handleDayChange(new Date());
     }
+    if (this.state.holidays.length === 0)
+    {
+      //let day = new Date();
+      //let year = day.getFullYear();
+      this.getHolidays(2021); // change year later
```

Comment:
Please update the hardcoded 2021 year. You can look at[ this example](https://www.w3schools.com/jsref/jsref_getfullyear.asp) to get the current year.